app/backend/src/clj/teet/admin/index_queries.clj

```diff
@@ -0,0 +1,41 @@
+(ns teet.admin.index-queries
+  (:require [teet.db-api.core :as db-api :refer [defquery]]
+            [datomic.client.api :as d]
+            [teet.transit :as transit])
+  (:import (java.util UUID)))
+
+(defn- get-index-data
+  "Returns the indexes data"
+  [db]
+  )
+
+(defn- with-bigdec-format [x]
+  (transit/with-write-options
+    {java.math.BigDecimal pr-str}
+    x))
+
+(def index-listing-attributes
+  [:db/id
+   :cost-index/name
+   :cost-index/type
+   :cost-index/unit
+   :cost-index/valid-from
+   {:cost-index/values
+    [:db/id
+     :index-value/month
+     :index-value/year
+     :index-value/value
+     ]}])
+
+(defquery :admin/indexes-data
+          {:doc "Pull information about the indexes"
+           :context {:keys [db] :as ctx}
+           :args {}
+           :project-id nil
+           :authorization {:admin/manage-indexes {}}}
+            (with-bigdec-format
+              (->> (d/q '[:find (pull ?e columns)
```

Comment:
instead of returning vector, you should return `{:index-data (->> (d/q ...))}` so the route's state will be a map as it will have other keys